sticker-graph/sticker_graph/model.py

```diff
@@ -143,27 +159,23 @@ def subword_reprs(self):
         # Convert strings to a byte tensor.
         #
         # Shape: [batch_size, seq_len, subword_len]
-        subword_bytes = tf.strings.unicode_decode(
-            self.subwords, input_encoding='UTF-8')
-        subword_bytes_padded = subword_bytes.to_tensor(
-            0)[:, :, :self.args.subword_len]
+        with tf.device("/cpu:0"):
```

Comment:
is this necessary for `map_fn`?

Reply:
On the GPU it's just too slow.

Reply:
I see, maybe put the subwords placeholder on the cpu too?

Reply:
Would be a useful thing to try. I'll make that a separate PR if that helps.